modular_zubbers/code/modules/antagonists/bloodsucker/structures/coffin.dm

```diff
@@ -221,10 +221,14 @@
 		if(!bloodsuckerdatum.my_clan)
 			user.balloon_alert("enter a clan!")
 			to_chat(user, span_notice("You must enter a Clan to rank up. Do it in the antag menu, which you can see by pressing the action button in the top left."))
-		else
+		if(bloodsuckerdatum.bloodsucker_level_unspent > 0)
```

Comment:
:P

Reply:
Code completely refactored. If you want to give it a look over, feel free to!